api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/PublishNodesEndpointApiModel.cs

```diff
@@ -14,48 +15,53 @@ namespace Microsoft.Azure.IIoT.OpcUa.Api.Publisher.Models {
     [DataContract]
     public class PublishNodesEndpointApiModel {
 
-        /// <summary> The Group the stream belongs to - DataSetWriterGroup. </summary>
+        /// <summary> The Group the stream belongs to - DataSetWriterGroup.</summary>
         [DataMember(Name = "dataSetWriterGroup", Order = 0,
             EmitDefaultValue = false)]
         public string DataSetWriterGroup { get; set; }
 
-        /// <summary> Id Identifier of the DataFlow - DataSetWriterId. </summary>
+        /// <summary> Id Identifier of the DataFlow - DataSetWriterId.</summary>
         [DataMember(Name = "dataSetWriterId", Order = 1,
             EmitDefaultValue = false)]
         public string DataSetWriterId { get; set; }
 
-        /// <summary> The Publishing interval for a dataset writer </summary>
+        /// <summary> The Publishing interval for a dataset writer.</summary>
```

Comment:
Let's specify whether this is in seconds or milliseconds.

Reply:
done

Reply:
I still do not see in the comment whether `DataSetPublishingInterval` is in seconds or milliseconds.